doc/source/_toc.yml

```diff
@@ -14,6 +14,13 @@ parts:
       - file: ray-overview/ray-libraries
         title: "Ecosystem"
 
+      - file: batch-inference/getting-started
```

Comment:
A good way to extend this to other CUJs would be to make our "Use Cases" section expandable and then have several cases such as this one there.